src/main/java/org/fundacionjala/fc/client/Authentication.java

```diff
@@ -0,0 +1,56 @@
+package org.fundacionjala.fc.client;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.apache.http.entity.ContentType;
+
+public class Authentication {
+
+    private static final String BASE_URL_API = "http://localhost:8080";
+    private static final String USERNAME_VAL = "admin2";
+    private static final String PASSWORD_VAL = "Admin123451";
+    private static final String USERNAME = "username";
+    private static final String PASSWORD = "password";
+    private static final String PATH = "/login";
+    private RequestSpecification request;
+
+    /**
+     * Constructor for Authentication.
+     */
+    public Authentication() {
+        request = RestAssured.given();
```

Comment:
Please consider moving these 3 statements to the `getRequestSpecification()` method.